### PR TITLE
Convert to a VS2017 project

### PR DESCRIPTION
**Pull request by droyad**

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "projects": [ "source" ],
```
Comment (johnsimons) [inline global.json]: How did u find out about "projects" ? It is not documented https://docs.microsoft.com/en-us/dotnet/articles/core/tools/global-json

Comment (johnsimons) [inline global.json]: Apparently u don't need it, see https://github.com/dotnet/docs/pull/996#issuecomment-285505275

Comment (droyad) [inline global.json]: My bad, copy pasta :)

File: build.cake [hunk 2]
```diff
@@ -2,243 +2,118 @@
 // TOOLS
 //////////////////////////////////////////////////////////////////////
 #tool "nuget:?package=GitVersion.CommandLine&version=4.0.0-beta0007"
-#addin "MagicChunks"
+#tool "nuget:?package=NUnit.ConsoleRunner"&version=3.6.1
```
Comment (johnsimons) [inline build.cake]: @droyad given that we have upgraded to xunit, I don't think we need this anymore

**Pull request merged**